scripts/reexport-omnivoice/codes/model_wrappers.py

```
@@ -0,0 +1,251 @@
+    (`torch.nn.utils.parametrizations.weight_norm`). Both add non-persistent state that
+    `torch.onnx.export` chokes on, so we fold the effective weight into a plain Parameter.
+
+    HiggsAudioV2TokenizerModel exposes `remove_weight_norm()` but it only covers the DAC
+    acoustic encoder/decoder; the HuBERT semantic model's `pos_conv_embed.conv` is
+    parametrized too, so we also walk every submodule generically.
+    """
+    if hasattr(module, "remove_weight_norm"):
+        try:
+            module.remove_weight_norm()
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)